app/models/inventory_item.rb

```diff
@@ -1,2 +1,9 @@
 class InventoryItem < ActiveRecord::Base
+  using(:grossman)
+  after_initialize :init
```

Comment:
OK Ill leave the branch open but close the push request as we may need to work on this later when Payroll/other apps are integrated with the gem.